modules/tensor_mechanics/include/materials/cohesive_zone_model/CZMComputeDisplacementJumpBase.h

```diff
@@ -35,24 +47,21 @@ class CZMComputeDisplacementJumpBase : public InterfaceMaterial
   /// Base name of the material system
   const std::string _base_name;
 
-  /// the interface normal in the undeformed configuration
-  const MooseArray<Point> & _normals;
-
```

Comment:
delete it because it was not used?

Reply:
We don't need this, as `_normals` is already provided in the `MaterialBase` base class.